src/main/java/io/github/thebusybiscuit/slimefun4/api/player/PlayerBackpack.java

```diff
@@ -71,7 +72,7 @@ public PlayerBackpack(@Nonnull PlayerProfile profile, int id, int size) {
         this.cfg = profile.getConfig();
         this.size = size;
 
-        cfg.setValue("backpacks." + id + ".size", size);
+        cfg.setValue(PlayerBackpack.backpacks + id + ".size", size);
```

Comment:
`PlayerBackpack` is redundant here, no need to specify the class you are already in.

Reply:
ff83a0183